samples/bot-consent-azuread/nodejs/src/static/LoginEnd.html

```diff
@@ -0,0 +1,94 @@
+<div class="font-semibold font-title">LoginEnd.html</div>
+<p>
+    <div>
+    </div>
+    <script src="https://statics.teams.microsoft.com/sdk/v1.5.2/js/MicrosoftTeams.min.js" crossorigin="anonymous"></script>
```

Comment:
Could you please update the sample to use latest SDK i.e. v2? 
Ref: https://docs.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/using-teams-client-sdk?tabs=javascript%2Cmanifest-teams-toolkit#whats-new-in-teamsjs-version-20

Reply:
@scottperham has sorted this Wajeed